cmd/kubeadm/app/phases/apiconfig/setupmaster.go

```diff
@@ -45,7 +45,13 @@ func attemptToUpdateMasterRoleLabelsAndTaints(client *clientset.Clientset) error
 		if n, err = client.Nodes().Get(node.GetHostname(""), metav1.GetOptions{}); err != nil {
```

Comment:
At this point an error is expected to happen several times (IIUC both apiserver can be available and node not registered yet)